components/partial/code/CodeExamples.tsx

```diff
@@ -144,19 +105,31 @@ export default function CodeExamples({
     (l) => getLanguageOptions(l, ctx.languageOptions) || { value: l, label: l }
   );
 
+  const { copied, copy } = useCopy(codeString);
+
+  const onlyOneExample = options.length === 1;
+
   return (
     <div>
-      <SwitcherBar>
-        <Select
-          autoWidth
-          variant="code"
-          options={options}
-          selectedValue={options.find((o) => o.value === selectedLanguage)}
-          setSelectedValue={({ value }) => setSelectedLanguage(value)}
-        />
-      </SwitcherBar>
-
-      <CodeBlock>{codeExampleHighlighted}</CodeBlock>
+      <CodeBlockBar>
+        {onlyOneExample ? (
+          <BarText>{activeLanguage}</BarText>
+        ) : (
+          <Select
+            autoWidth
+            variant="code"
+            options={options}
+            selectedValue={options.find((o) => o.value === activeLanguage)}
+            setSelectedValue={({ value }) => setSelectedLanguage(value)}
+          />
```

Comment:
I noticed that one of the options is `bash`, but we have it as `raw` in the dropdown that we have in the top nav. Can we switch one to the other so it's more consistent?

Reply:
Yeah nice catch,

I updated the value in `LANGUAGE_OPTIONS` for now to be `bash`, but we may want to update the .mdx files to use ```raw. cc @spencerhunter